design/design-doc-stacks.md

```diff
@@ -103,32 +109,32 @@ status:
 
 ## `Extension` CRD
 
-The `Extension` CRD serves as a record of an installed extension (a custom controller and its CRDs).
+The `Extension` CRD serves as a record of an installed Stack (a custom controller and its CRDs).
```

Comment:
Do we plan to rename the `Extension` CRD to `Stack` at some point, or will this remain unchanged?

Reply:
I noticed we do rename it to `Stack` below, so I think we need to make sure these are consistent.

Reply:
I've corrected the inadvertent `Kind: Stack`. Thanks.

A follow-up PR will pick up code renaming of the CRD (or choose not to, a benefit of this PR being separate and less aggressive).